node/resolvers/Mutations/Users.ts

```diff
@@ -183,26 +183,29 @@ const Users = {
       const result = await storefrontPermissionsClient.getRole(user.roleId)
       const roleSlug = result?.data?.getRole?.slug
 
-      let permitted = false
+      const canImpersonateUsersWithCostCenterPermission =
+        !roleSlug.includes('admin') &&
```

Comment:
I also wonder why we need to check if role is not `admin`.  Admins can't impersonate "with cost centers"? But can impersonate on the other cases?

Reply:
I kept the implementation, but now, thinking more, let's remove all roles and just use permissions.